travis/before_install.sh

```diff
@@ -1,38 +1,25 @@
 #!/bin/bash
 
+set -e
+
 echo "Running before_install"
-echo "SUITE: ${SUITE}"
 
-"echo 'gem: --no-ri --no-rdoc' > ~/.gemrc"
+# Assert that the given SUITE is a known one
+case "$SUITE" in
+    rspec|rubocop|cucumber)
+        echo "SUITE: ${SUITE}"
+        ;;
+    *)
+        echo -e "Error: empty or unknown SUITE: $SUITE\n"
```

Comment:
You can ditch the `-e` and remove the `\n` in the end of the string. By default `echo` prints a newline.